src/handwritten/_install_daqmx.py

```diff
@@ -117,49 +152,89 @@ def _multi_access_temp_file(
                 ) from e
 
 
-def _load_data(json_data: str) -> Tuple[Optional[str], Optional[str]]:
+@contextlib.contextmanager
+def _multi_access_temp_folder(*, delete: bool = True) -> Generator[str, None, None]:
+    """
+    Context manager for creating a temporary folder.
+
+    """
+    try:
+        temp_folder = tempfile.TemporaryDirectory()
```

Comment:
TemporaryDirectory is a context manager, so you don't need try/except. It will automatically clean up.

```python
with tempfile.TemporaryDirectory() as temp_folder:
    ...
```

Do you even need to wrap it with a function?